presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/CanonicalizeExpressionRewriter.java

```diff
@@ -107,5 +111,29 @@ public Expression rewriteExtract(Extract node, Void context, ExpressionTreeRewri
 
             throw new UnsupportedOperationException("not yet implemented: " + node.getField());
         }
+
+        @Override
+        public Expression rewriteFunctionCall(FunctionCall node, Void context, ExpressionTreeRewriter<Void> treeRewriter)
+        {
+            if (!rewriteFns.contains(node.getName())) {
+                return null;
+            }
+            ImmutableList.Builder<Expression> fnArgs = ImmutableList.builder();
```

Comment:
same comment. Don't use short forms for variable names.

Reply:
made changes